app/main.py

```diff
@@ -1,2 +1,77 @@
+from __future__ import annotations
```

Comment:
The import statement `from __future__ import annotations` is correct, but ensure that your Python version supports this feature. This import is used to postpone the evaluation of type annotations, which can be beneficial for forward references.